src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```diff
@@ -71,7 +71,19 @@ services:
         class: PrestaShop\PrestaShop\Adapter\Product\Image\ProductImagePathFactory
         arguments:
             - "@=service('prestashop.adapter.legacy.configuration').getBoolean('PS_LEGACY_IMAGES')"
+            - ''
+            - !php/const _PS_PROD_IMG_
             - !php/const _PS_TMP_IMG_DIR_
+            - '@=service("prestashop.adapter.legacy.context").getContext().language.iso_code'
+
+    prestashop.adapter.product.image.product_image_url_factory:
```

Comment:
Mmmm, love this one ^^

Reply:
of course you do, it was your idea :sweat_smile:

Reply:
now this makes me look so self-centered 😅